rizkyhidayatullah_211011401221/tugas2.doc

```diff
@@ -0,0 +1,27 @@
+FUNGSIIONAL
+Sistem order di meja
+-User dapat memilih untuk memesan makanan secara dine-in atau delivery.
+-Pesanan yang dipilih akan dikirimkan ke dapur secara otomatis untuk diproses oleh koki.
+-User dapat melihat daftar menu makanan yang tersedia di restoran.
+-User dapat melakukan pencarian menu untuk menemukan item makanan yang diinginkan.
+-User dapat mengedit pesanan mereka sebelum mengirimkan pesanan ke dapur.
+-Ketika user memilih menu makanan, mereka dapat mengklik tombol tambah untuk menambahkan item ke pesanan mereka.
+-User dapat menambahkan opsi split bill jika mereka makan bersama dalam grup.
+-User dapat memilih metode pembayaran yang diinginkan.
+-Pelanggan diberikan opsi untuk memberikan tip secara tunai kepada pelayan atau staf restoran setelah menerima layanan.
+-Sistem dapat memberikan rekomendasi menu berdasarkan pesanan sebelumnya atau popularitas item.
+-User dapat memberikan penilaian dan ulasan untuk makanan yang mereka pesan langsung melalui tablet.
+-Sistem menyediakan informasi nutrisi untuk setiap menu, membantu pelanggan membuat pilihan makanan yang lebih sehat.
+
+sistem untuk koki
+-Koki dapat mengatur prioritas pesanan berdasarkan waktu dan jenis pesanan.
+-Sistem akan melakukan pembaharuan stok bahan secara real-time.
+-Sistem dapat menghasilkan laporan harian yang mencakup jumlah pemesanan, waktu persiapan, dan penggunaan bahan.
+-Sistem akan memberikan peringatan jika stok bahan sudah rendah.
+
+NON-FUNGSIONAL
+-Antarmuka pengguna harus intuitif dan mudah digunakan oleh pengguna dengan berbagai tingkat keahlian teknologi.
+-Sistem hanya bisa diakses di restoran atau di rumah menggunakan aplikasis
+-Sistem harus melindungi data pelanggan dan informasi pembayaran dengan baik.
+-Sistem harus mampu merespon input pengguna dengan cepat.
+-Pemrosesan pesanan harus dilakukan dalam waktu kurang dari 5 detik setelah user menekan tombol submit.
```

Comment:
Good job👍
Sedikit tambahan, bisa ditambah fungsional bahwa tiap pesanan di dalam sistem order sudah memiliki nomor meja sendiri agar tidak tertukar dengan pesanan di meja lain. Non-fungsional bisa ditambahkan bahwa sistemnya berjalan di tablet android. Bisa lebih rapi juga kalau disajikan menggunakan tabel dan ada entity usernya.
But after all, sudah bagus dan sudah bisa membuat analisis dari apa yang kemarin dibahas. Keep up the good work.